packages/eslint-config-nori/index.js

```diff
@@ -194,7 +194,7 @@ module.exports = {
           'error',
           { selector: 'class', format: ['PascalCase'] },
           {
-            selector: ['function', 'parameter'],
+            selector: ['function'],
```

Comment:
removing the 'parameter' selector here allows our existing parameter rule to be used for detecting linting errors

![image](https://user-images.githubusercontent.com/7808563/213574452-469d14e8-e81b-4f78-8390-7d87eb402ff9.png)

Reply:
The existing rule allows underscore prefixes, which also allows '_' as a parameter name